packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.8.7+2
 
+* Example file showcases how to display an error placeholder whenever an unsupported image type is selected.
```

Comment:
We generally describe the change with a verb. E.g., "Adds handling of unsupported image types to the example."